routes.schema.json

```diff
@@ -18,6 +18,29 @@
         "type": "string"
       }
     },
+    "optionalBackendDependencies": {
+      "type": "object",
+      "description": "A list of backend modules that may enable optional functionality in this frontend module if available and the corresponding required versions."
+      "additionalProperties": {
+        "oneOf": [
+          {
+            "type": "string"
+          },
+          {
+            "type": "object",
+            "properties": {
+              "version": {
+                "type": "string",
+                "description": "A SemVer range specifying the allowed versions"
+              }
```

Comment:
Are we not adding the `feature` property here too?

Reply:
Not for now... It doesn't (yet) do anything. I'll add it in when it does.